src/components/SiteList.js

```diff
@@ -16,26 +17,27 @@ class SiteList extends Component {
     this.setState({ sites });
   }
 
-  siteSelectHandler(id) {
-    this.props.history.push({ pathname: '/' + id });
-  }
+  siteSelectHandler = selectedOption => {
```

Comment:
I'm happy that you're using this patten, but...

Can you tell me the difference between fat arrow syntax:
```js
specialFn = param => { .. } 
```

and the classic syntaxes: 
```js
specialFn (params) { .. } 
```

Reply:
if I do a classic syntax, I will have to bind this to class base component. Other option, use arrow function, different styles but I do prefer this approach.

Reply:
I also prefer fat arrow as well.